Formula/g/gstreamer.rb

```diff
@@ -52,66 +52,92 @@ class Gstreamer < Formula
 
   depends_on "bison" => :build
   depends_on "cargo-c" => :build
-  depends_on "gitlint" => :build
+  depends_on "gettext" => :build
   depends_on "gobject-introspection" => :build
   depends_on "meson" => :build
   depends_on "nasm" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
-  depends_on "yasm" => :build
+  depends_on "aom"
   depends_on "cairo"
   depends_on "dav1d"
   depends_on "faac"
   depends_on "faad2"
   depends_on "fdk-aac"
   depends_on "ffmpeg@6"
   depends_on "flac"
-  depends_on "gettext"
+  depends_on "gdk-pixbuf"
   depends_on "glib"
-  depends_on "glib-networking"
```

Comment:
Didn't see any linkage and looks specific to `glib` subproject - https://gitlab.freedesktop.org/gstreamer/gstreamer/-/commit/237f5cc4ee8a20a51d8191e5c96c2b0efa268b5c